tests/pytest/core/test_analytics.py

```diff
@@ -80,3 +80,49 @@ def test_Event_update_user_properties(app_request):
 
     assert key in event.user_properties
     assert event.user_properties[key] == value
+
+
+@pytest.mark.django_db
+def test_ViewedPageEvent_sets_default_event_properties(app_request, mocker):
+    update_spy = mocker.spy(ViewedPageEvent, "update_event_properties")
+
+    ViewedPageEvent(app_request)
+
+    utm_codes = ["utm_campaign", "utm_source", "utm_medium", "utm_content", "utm_id"]
+    for utm_code in utm_codes:
+        assert utm_code in update_spy.call_args.kwargs
+        assert update_spy.call_args.kwargs.get(utm_code) is None
```

Comment:
These asserts are good, in that they check that the `update_*_properties` function was called. I think of this like the "what" of the changes you made to the `ViewedPageEvent` class.

But we still want to check the result, or the "how" of your changes -- i.e. how the changes manifested when using the now updated `ViewedPageEvent` class.

We should see these new keys in the `*_properties` attributes on the event:

```python
event = ViewedPageEvent(app_request)

assert event.event_properties["utm_campaign"] is None
```

This asserts 2 things: 

1. `event.event_properties` has a key `"utm_campaign"` (otherwise a `KeyError` would be raised and the assert would fail)
2. the value of that key is `None`

These are the 2 expected outcomes of the changes you made when constructing a `ViewedPageEvent` with an `app_request` that has no UTM information on it.